### ISSUE TITLE
Incorrect loss normalization when masking padded actions in ACT (mean over padded horizon)

### ISSUE THREAD

**Issue by harshab1701** [huggingface/lerobot#3353]
### Ticket Type

🐛 Bug Report (Something isn't working)

### Environment & System Info

```Shell

```

### Description

The current loss computation in ACT applies a mask to padded actions but still performs a global .mean() reduction. This results in incorrect normalization when sequences contain padding.

Specifically, padded action entries are zeroed out before reduction, but they are still included in the denominator when computing the mean. This causes the loss to be underestimated for sequences with more padding i.e., 60 to 70% padding.

### Context & Reproduction

**Current implementation (line 114 of modelling_act)**
https://github.com/huggingface/lerobot/blob/main/src/lerobot/policies/act/modeling_act.py#L144
`l1_loss = (
    F.l1_loss(batch[ACTION], actions_hat, reduction="none")
    * ~batch["action_is_pad"].unsqueeze(-1)
).mean()`

**Issue**:

- action_is_pad correctly masks padded timesteps
- however, .mean() still divides by the total number of elements (including padded ones)
- this effectively dilutes the contribution of valid (non-padded) actions

**Expected behavior**

The loss should be normalized only over valid (non-padded) action entries, not over the full padded horizon.

A correct formulation would be:

```
abs_err = F.l1_loss(batch[ACTION], actions_hat, reduction="none")

valid_mask = ~batch["action_is_pad"].unsqueeze(-1)
num_valid = valid_mask.sum() * abs_err.shape[-1]

l1_loss = (abs_err * valid_mask).sum() / num_valid.clamp_min(1)
```

### Relevant logs or stack trace

```Shell

```

### Checklist

- [x] I have searched existing tickets to ensure this isn't a duplicate.
- [x] I am using the latest version of the `main` branch.
- [x] I have verified this is not an environment-specific problem.

**Comment by whats2000** [comment]
@harshab1701 I have reproduced the issue! We will patch it right away!
